h2o-py/h2o/explanation/_explain.py

```diff
@@ -2551,7 +2553,7 @@ def learning_curve_plot(
 
     selected_timestep_value = None
     if "number_of_trees" == timestep:
-        selected_timestep_value = model.actual_params["ntrees"]
+        selected_timestep_value = model.summary()["number_of_trees"][0]
```

Comment:
The `actual_params["ntrees"]` doesn't show the actual number of trees used for early stopping but it used to.